File: szavazas-core/src/main/java/hu/kdea/szavazas/ballotprocessor/BallotResult.java
package hu.kdea.szavazas.ballotprocessor;

import hu.kdea.szavazas.ballotprocessor.common.CellPositionData;
import java.util.List;

public record BallotResult(String raw, int numSupport, int numRows, List<CellPositionData> xCells) {
}

File: test-utils/src/main/java/hu/kdea/szavazas/BallotResult.kt
// app/src/main/java/hu/kdea/szavazas/BallotResult.kt
package hu.kdea.szavazas

data class BallotResult(
    val raw: String,
    val numSupport: Int,
    val numRows: Int,
    val xCells: List<Pair<Int, Int>>
)